Sources/Views/SubscriptionConfirmation.swift

```diff
@@ -147,18 +167,30 @@ updateSeats()
               ])
           ],
           [
-            "You can add additional team members at anytime from your account page."
+            """
+You must have at least two seats for your team subscription. You can add additional team members at anytime
+from your account page.
+"""
           ]
         )
       ]
     )
   ]
 }
 
-private func teamOwner(_ currentUser: User) -> Node {
+private func teamOwner(currentUser: User, subscribeData: SubscribeConfirmationData) -> Node {
+  guard subscribeData.isOwnerTakingSeat else {
+    return input([
+      name(SubscribeData.CodingKeys.isOwnerTakingSeat.rawValue),
+      `type`(.hidden),
+      value("false")
+      ])
```

Comment:
this won't render the owner row if they aren't taking a seat. this is used when redirecting back to this page after a failed subscription.